### PR TITLE
separate default builtins for platforms

### PR DESCRIPTION
**Pull request by restrry**
issue https://github.com/babel/babel-preset-env/issues/220

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -161,6 +161,14 @@ export const transformIncludesAndExcludes = (opts) => ({
   builtIns: opts.filter((opt) => opt.match(/^(es\d+|web)\./))
 });
 
+function getPlatformSpecificDefaultFor(targets) {
+  const targetNames = Object.keys(targets);
+  const isAnyTarget = !targetNames.length;
+  const isWebTarget = targetNames.some((name) => name !== "node");
+
+  return (isAnyTarget || isWebTarget) ? defaultInclude : [];
```
Comment (restrry) [inline src/index.js]: should we rename it?

Comment (existentialism) [inline src/index.js]: Maybe `defaultInclude` -> `defaultWebIncludes`?

Comment (hzoo) [inline src/index.js]: yeah since `import defaultInclude from "./default-includes";`

File: src/index.js [hunk 2]
```diff
@@ -161,6 +161,14 @@ export const transformIncludesAndExcludes = (opts) => ({
   builtIns: opts.filter((opt) => opt.match(/^(es\d+|web)\./))
 });
 
+function getPlatformSpecificDefaultFor(targets) {
+  const targetNames = Object.keys(targets);
+  const isAnyTarget = !targetNames.length;
+  const isWebTarget = targetNames.some((name) => name !== "node");
```
Comment (yavorsky) [inline src/index.js]: maybe check for `electron` also?

Comment (hzoo) [inline src/index.js]: but you would want to include it for electron/uglify right?

Comment (yavorsky) [inline src/index.js]: If we have node + electron or node + uglify, isWebTarget will be true. But we don't need to include it in these cases ?

**Pull request merged**